### PR TITLE
Extra v6.7 ChangeLog addition

### PR DESCRIPTION
**Pull request by kinkie**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -2,6 +2,7 @@ Changes in squid-6.7 (4 Feb 2024)
 
 	- Bug 5274: Successful tunnels logged as TCP_TUNNEL/500
 	- Fix crash when NTLM and Negotiate helpers are queried with no HTTP request
+	- Fix memory leak on SslBump certificates with Authority Key Identifier extension
```
Comment (rousskov) [inline ChangeLog:5]: IIRC, we do not normally use an "SslBump certificate" term. I do not insist on any changes or any specific fix here, but can suggest this variation:

```suggestion
	- Fix SslBump memory leak when mimicking certificates with Authority Key Identifier
```

or, slightly more detailed/longer:

```suggestion
	- Fix SslBump memory leak when mimicking certificates with Authority Key Identifier extension
```

**Comment by rousskov** [comment]
> kinkie requested a review from rousskov 11 minutes ago

FYI: I cannot approve this PR until v6 is in sync with the proposed ChangeLog changes. That is why I added an S-waiting-for-PR label...

**Review by yadij — APPROVED** [review]

**Comment by kinkie** [comment]
@rousskov the dependent PR should be okay now; would you mind approving? We are aiming for releasing on Sunday

**Comment by rousskov** [comment]
> Alex: I cannot approve this PR until v6 is in sync with the proposed ChangeLog changes. 

> Francesco: the dependent PR should be okay now; would you mind approving?

The "dependent PR" state have never influenced my reluctance to approve this PR. As I tried to explain in the quoted comment, ChangeLog for v6 (i.e. the subject of this master PR) should match v6 branch. It _still_ does not: This mater PR got merged with entries that do not reflect the current v6 state (as of v6 commit a0dc688).

I can only continue to beg you to change this aspect of the release process that creates completely unnecessary delays, increases overheads, and puts me in an awkward position of having to pick between an approval of incorrect changes and an increased risk of being perceived as uncooperative.

**Comment by kinkie** [comment]
> I can only continue to beg you to change this aspect of the release process

We are thinking about it and assessing alternatives and pros and cons, but right now we have a planned release in 2 days so we need to stick with what we have

**Comment by squidadm** [comment]
queued for backport to v6

**Pull request closed without merging**